app/components/treatment/Treatment.js

```diff
@@ -1,13 +1,21 @@
 import React, { Component } from 'react'
```

Comment:
This page will throw an error if you try to load it directly/reload. We should probably add logic in the `enterHook` inside `main.js` to handle loading the appropriate data for this case